gravitee-gateway-standalone/gravitee-gateway-standalone-container/src/test/java/io/gravitee/gateway/standalone/PostContentGatewayTest.java

```diff
@@ -148,4 +148,29 @@ public void call_case3_raw() throws Exception {
             assertTrue(false);
         }
     }
+
```

Comment:
Maybe test that a request with empty body but no Transfer-Encoding header is forwarded to upstream without being chunked (basically : please add also the opposite of your current test) ?

Reply:
Added

Reply:
lgtm